packages/@react-spectrum/tabs/stories/Tabs.stories.js

```diff
@@ -155,174 +194,184 @@ storiesOf('Tabs', module)
-          </>
-        }>
-        <Content margin="size-160">
-          <Heading>Calendar</Heading>
-          <Text>
-            Dolore ex esse laboris elit magna esse sunt. Pariatur in veniam Lorem est occaecat do magna nisi mollit ipsum sit adipisicing fugiat ex. Pariatur ullamco exercitation ea qui adipisicing.
-            Id cupidatat aute id ut excepteur exercitation magna pariatur. Mollit irure irure reprehenderit pariatur eiusmod proident Lorem deserunt duis cillum mollit. Do reprehenderit sit cupidatat quis laborum in do culpa nisi ipsum. Velit aliquip commodo ea ipsum incididunt culpa nostrud deserunt incididunt exercitation. In quis proident sit ad dolore tempor. Eiusmod pariatur quis commodo labore cupidatat cillum enim eiusmod voluptate laborum culpa. Laborum cupidatat incididunt velit voluptate incididunt occaecat quis do.
-            Consequat adipisicing irure Lorem commodo officia sint id. Velit sit magna aliquip eiusmod non id deserunt. Magna veniam ad consequat dolor cupidatat esse enim Lorem ullamco. Anim excepteur consectetur id in. Mollit laboris duis labore enim duis esse reprehenderit.
-          </Text>
-        </Content>
-      </Item>
-      <Item
-        key="bookmark"
-        textValue="Bookmark"
-        title={
-          <>
-            <Bookmark />
-            <Text>Bookmark</Text>
-          </>
-        }>
-        <Content margin="size-160">
-          <Heading>Bookmark</Heading>
-          <Text>
-            Dolore ex esse laboris elit magna esse sunt. Pariatur in veniam Lorem est occaecat do magna nisi mollit ipsum sit adipisicing fugiat ex. Pariatur ullamco exercitation ea qui adipisicing.
-            Id cupidatat aute id ut excepteur exercitation magna pariatur. Mollit irure irure reprehenderit pariatur eiusmod proident Lorem deserunt duis cillum mollit. Do reprehenderit sit cupidatat quis laborum in do culpa nisi ipsum. Velit aliquip commodo ea ipsum incididunt culpa nostrud deserunt incididunt exercitation. In quis proident sit ad dolore tempor. Eiusmod pariatur quis commodo labore cupidatat cillum enim eiusmod voluptate laborum culpa. Laborum cupidatat incididunt velit voluptate incididunt occaecat quis do.
-            Consequat adipisicing irure Lorem commodo officia sint id. Velit sit magna aliquip eiusmod non id deserunt. Magna veniam ad consequat dolor cupidatat esse enim Lorem ullamco. Anim excepteur consectetur id in. Mollit laboris duis labore enim duis esse reprehenderit.
-          </Text>
-        </Content>
-      </Item>
+      <TabList>
+        <Item key="dashboard" textValue="Dashboard">
+          <Dashboard />
+          <Text>Dashboard</Text>
+        </Item>
+        <Item key="calendar" textValue="Calendar">
+          <Calendar />
+          <Text>Calendar</Text>
+        </Item>
+        <Item key="bookmark" textValue="Bookmark">
+          <Bookmark />
+          <Text>Bookmark</Text>
+        </Item>
+      </TabList>
+      <TabPanels>
+        <Item key="dashboard">
```

Comment:
The TabPanel items will throw a warning here since there isn't a `textValue` provided. I wonder if we should modify `Item.ts` so that a `textValue` isn't explicitly required since the TabList and TabPanel items don't support typeahead anyways.